.github/workflows/tests-ln-dlc-node.yml

```diff
@@ -1,60 +0,0 @@
-name: Slow ln-dlc-node tests
-
-on:
-  push:
-    branches:
-      - "staging"
```

Comment:
as an alternative, it might be easier to just run add `-main` branch here too? so they are being run after merging, instead of on all open PRs?

Reply:
Well, my motivation is to see the effect of every push on all PRs. Rather than having to wait for it to merge.